### DISCUSSION TITLE
Some results I can't understand about pvefficiency_adr and df['p_mp']

### DISCUSSION THREAD

**Discussion by Typezero-shanxi** [pvlib/pvlib-python#2093] [Q&A] [answered]
When I run the official case "Simulating the DC output of a PV system using the ADR module efficiency model". I found the result graph as below
1. The first graph shows that the read efficiency of the PV panels is greater than 1. I would like to know if this is normal and if the official documentation and citations explain this phenomenon.
2. In addition, the second graph shows that the output of a PV panel with a rated capacity of 5kw can exceed 5kw at some times.

I hope that seniors and teachers who understand the reasons for this can give some guidance, thank you!
![Figure_1](https://github.com/pvlib/pvlib-python/assets/156188273/a08bad0d-98e8-405e-b57c-1c422ac8e8bc)

![Figure_2](https://github.com/pvlib/pvlib-python/assets/156188273/fe786107-baba-4680-b128-22f55d097187)

**Comment by kandersolar** [comment, marked as answer]
Both of these are normal.  The relative efficiency is _relative_ to efficiency at standard test conditions (STC), i.e. 1000 W/m^2 irradiance and 25 degrees C.  The nominal rating of a PV module is measured at STC as well.  The rating of a PV module is not its _maximum_ capacity but rather a measure of how much power it produces at a particular set of convenient conditions.  

If irradiance is higher than 1000 W/m^2, the power can certainly exceed its nominal rating.  A PV module produces more power at an irradiance of 1100 W/m^2 than it does at 1000 W/m^2.  

And since PV devices become more efficient at lower temperature (see how the high efficiency points in your plot are all blue?), then relative efficiency can exceed 1.0 at low temperature.  Search "PV temperature coefficient" to learn more about this.

**Reply by Typezero-shanxi** [reply]
Thanks for the Q&A, my unfamiliarity with PV systems caused a misunderstanding. I will look up "PV temperature coefficient" as you suggested. In addition, I would like to know if you know if there is some formula or coefficient for quantitative expression of the first question.
